.vscode/settings.json

```diff
@@ -1,3 +1,6 @@
 {
```

Comment:
@thomaseizinger you are the only one using vscode for that. I am not sure it makes sense for this folder to be here.

Reply:
We can check in the configuration of other editors as-well. I wouldn't object :)